src/vivarium/framework/randomness/manager.py

```diff
@@ -74,6 +76,10 @@ def setup(self, builder: Builder) -> None:
         pop_size = builder.configuration.population.population_size
         map_size = max(map_size, 10 * pop_size)
         self._key_mapping_ = IndexMap(self._key_columns, map_size)
+        if builder.configuration.randomness.rate_conversion_type is not None:
```

Comment:
Rather than have this if/else, you can just set the default value to be "linear" rather than None

Reply:
Wouldn't this make it not a major release since the key would be required then?

Reply:
Updated